addons/core/addon/helpers/doc-url.js

```diff
@@ -1,20 +1,23 @@
-import { helper } from '@ember/component/helper';
+import Helper from '@ember/component/helper';
+import { getOwner } from '@ember/application';
 import { assert } from '@ember/debug';
-import config from 'ember-get-config';
 
-export default helper(function docUrl(params /*, hash*/) {
-  const baseURL = config.documentation.baseURL;
-  const docKey = params[0] || '';
-  const configuredPath = config.documentation.topics[docKey];
-  if (docKey) {
-    assert(
-      `
-      Documentation for "${docKey}" could not be found. Please ensure that
-      this key exists under "documentation" in your app config.
-    `,
-      configuredPath
-    );
+export default class extends Helper {
+  compute(params) {
+    const config = getOwner(this).resolveRegistration('config:environment');
+    const baseURL = config.documentation.baseURL;
+    const docKey = params[0] || '';
+    const configuredPath = config.documentation.topics[docKey];
+    if (docKey) {
+      assert(
+        `
+        Documentation for "${docKey}" could not be found. Please ensure that
```

Comment:
Asking with future improvements in mind: there is a possibility to include this text to the translations files? or does not make sense to add it to translations?

Reply:
This is an error message for developers, not end users.  Currently, only strings presented to the user are intended for translation.